src/metabase/events.clj

```diff
@@ -155,3 +159,21 @@
    ;; and is important for distinguishing view events triggered when pinned cards are 'viewed'
    ;; when a user opens a collection.
    :context      (:context object)})
+
+(def ^:private async-publish-event-timeout 10000)
+
+(defn publish-event-async!
```

Comment:
If it is the case that we want this as only a short-term fix, maybe we should include that in the code somehow? 
Perhaps log a warning? Or write a ;;todo: remove this in favor of a more complete synchronous solution. Or something.

Reply:
I'm fine with a TODO.

Reply:
TODO message pushed.